argocd error while deploying kubernetes cluster: name cannot be provided when a selector is specified

```
(You should delete the initial secret afterwards as suggested by the Getting Started Guide: https://argo-cd.readthedocs.io/en/stable/getting_started/#4-login-using-the-cli)
docker run --rm -v ~/.kube:/kube -e KUBECONFIG=/kube/config --network=host -v`pwd`:/host -w /host -u root --entrypoint /bin/sh bitnami/kubectl:latest -c  '\
        kubectl apply -f ./kcl-cmp.yaml && \
        kubectl -n argocd patch deploy/argocd-repo-server -p "`cat ./patch-argocd-repo-server.yaml`" && \
        kubectl wait --for=condition=ready pod -n argocd -l app.kubernetes.io/name=argocd-repo-server --timeout=600s \
        sleep 20 \
'
configmap/kcl-plugin-config created
deployment.apps/argocd-repo-server patched
error: name cannot be provided when a selector is specified
make: *** [Makefile:55: argocd] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -59,6 +59,14 @@ argocd:
 	$(KUBECTL_RUN) '\
 		kubectl apply -f ./kcl-cmp.yaml && \
 		kubectl -n argocd patch deploy/argocd-repo-server -p "`cat ./patch-argocd-repo-server.yaml`" && \
+		while :; do \
+			kubectl -n argocd get pods -l app.kubernetes.io/name=argocd-repo-server --field-selector=status.phase=Running | grep argocd-repo-server || { \
+				echo -n .; \
+				sleep 1; \
+				continue; \
+			}; \
+			break; \
+		done ; \
 		kubectl wait --for=condition=ready pod -n argocd -l app.kubernetes.io/name=argocd-repo-server --timeout=600s \
 	'
 

